src/db.rs

```diff
@@ -70,12 +70,13 @@ impl DB {
         Ok(())
     }
 
-    pub fn migrate<P: AsRef<Path>>(&mut self, path: P) -> Result<()> {
-        if !self.dirs.is_empty() {
-            bail!(
-                "To prevent conflicts, you can only migrate from z with an empty \
-                 zoxide database!"
-            );
+    pub fn migrate<P: AsRef<Path>>(&mut self, path: P, merge: bool) -> Result<()> {
+        if !self.dirs.is_empty() && !merge {
+            bail!(format!(
```

Comment:
For future reference, the `bail!` macro also does formatting, so the `format!` there is redundant. What I meant, though, is that something like this would be nice:


```rust
            bail!(
                "To prevent conflicts, you can only migrate from z with an empty zoxide database!
If you wish to merge the two, specify the `--merge` flag."
            );
```

Reply:
I personally am not a fan of multiline strings that de-indent, but I do agree that it looks nicer than both a `format!` (or just `"{}\n{}"`) and `\n\`.